### PR TITLE
NIB: Friend assemblies link

### PR DESCRIPTION
**Pull request by ChrisMaddock**
## Summary

Relates to https://github.com/dotnet/docs/issues/1787. (The two articles listed previously look to have now been merged.)

### CODE REVIEW

File: xml/System.Security.Permissions/StrongNameIdentityPermissionAttribute.xml [hunk 1]
```diff
@@ -52,7 +52,7 @@
...
   
- The <xref:System.Security.Permissions.StrongNameIdentityPermissionAttribute> attribute can be used to define strong-name requirements for access to public members at the assembly level. In the .NET Framework version 2.0 and later, you can also use the <xref:System.Runtime.CompilerServices.InternalsVisibleToAttribute> attribute to specify that all nonpublic types in that assembly are visible to another assembly. For more information, see [Friend Assemblies](https://msdn.microsoft.com/library/df0c70ea-2c2a-4bdc-9526-df951ad2d055) or [NOT IN BUILD: Friend Assemblies (Visual Basic)](https://msdn.microsoft.com/library/80e7a33a-ca91-450b-a00e-c5a7986e228c).  
+ The <xref:System.Security.Permissions.StrongNameIdentityPermissionAttribute> attribute can be used to define strong-name requirements for access to public members at the assembly level. In the .NET Framework version 2.0 and later, you can also use the <xref:System.Runtime.CompilerServices.InternalsVisibleToAttribute> attribute to specify that all nonpublic types in that assembly are visible to another assembly. For more information, see [Friend Assemblies](/docs/standard/assembly/friend-assemblies).
```
Comment (rpetrusha) [inline xml/System.Security.Permissions/StrongNameIdentityPermissionAttribute.xml]:
```suggestion
 The <xref:System.Security.Permissions.StrongNameIdentityPermissionAttribute> attribute can be used to define strong-name requirements for access to public members at the assembly level. In the .NET Framework version 2.0 and later, you can also use the <xref:System.Runtime.CompilerServices.InternalsVisibleToAttribute> attribute to specify that all nonpublic types in that assembly are visible to another assembly. For more information, see [Friend Assemblies](~/docs/standard/assembly/friend-assemblies).  
```

**Comment by ChrisMaddock** [comment]
Gaaaah, tildes are required here? I thought tildes were discouraged everywhere.

One day I'll memorise the many whims of creating links on MS docs! 😅

**Comment by rpetrusha** [comment]
I still haven't memorized the many whims of creating links either, @ChrisMaddock. The tildes are used for cross-repo linking within our repos. So a link from the API reference to docs uses the tilde, as do links to the samples repo.

**Comment by rpetrusha** [comment]
And of course, when I suggested the change, I forgot that the file extension needs to be part of the URL. I've added it, and I'll approve and merge once the build completes, @ChrisMaddock.

**Comment by MSDN-WhiteKnight** [comment]
Links without tildes also work, but these are site-relative links, not repo-related. So for this article the correct syntax should be `/dotnet/standard/assembly/friend-assemblies`.

**Comment by rpetrusha** [comment]
I'll merge your PR now, @ChrisMaddock.

**Pull request merged**